agent/src/agent/pipeline/repository.py

```diff
@@ -68,3 +68,17 @@ def count_by_streamsets() -> Dict[int, int]:
 
 def get_by_streamsets_id(streamsets_id: int) -> List[Pipeline]:
     return session().query(Pipeline).filter(Pipeline.streamsets_id == streamsets_id).all()
+
+
+def add_deleted_pipeline_id(pipeline_id: str):
```

Comment:
Why you did not create an orm entity for deleted_pipelines table?

Reply:
just to keep it simple, I thought there's no need in creating a new class just to pass it's instances to the repository